packages/typescript/CHANGELOG.md

```diff
@@ -6,22 +6,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+### Added
+- Add more setup instructions to `@metamask/typescript-eslint-config` README ([#209](https://github.com/MetaMask/eslint-config/pull/209))
```

Comment:
Nit: We can omit this; it's referenced already in the sub-point for 250. We only need to reference changes that impact users of this package.

```suggestion
```